auth/store.go

```diff
@@ -706,6 +706,10 @@ func (as *authStore) isOpPermitted(userName string, revision uint64, key, rangeE
 		return nil
 	}
 
+	if userName == "" {
+		return ErrUserEmpty
```

Comment:
if username is empty, shouldnt line 721 return nil and we get permissiondenied at line724?